src/components/streamSelect/StreamSelect.css

```diff
@@ -25,19 +27,26 @@
   border-color: #30183e;
 }
 
-.stream-select-wrap button {
-  margin-left: 5px;
-  height: 25px;
-}
 
 .stream-select-wrap {
-  width: 100%;
   height: 100%;
   display: flex;
   justify-content: center;
   align-items: center;
 }
 
+.options-bar::before{
+  content:"";
+  margin-right:auto;
+  visibility: hidden;
+  width:225px;
```

Comment:
Wouldn't it be better to make this (and all related and similar elements) responsive? E.g: Just add some padding and and use/inherit the content width.

Reply:
> Wouldn't it be better to make this (and all related and similar elements) responsive? E.g: Just add some padding and and use/inherit the content width.

I did this because `.options-bar::before` and `.latency-select-wrap` need to be the same width in order for the center element to stay in the center. I could use a percentage width in order to make it more responsive?

Reply:
You mean because you're aligning the left sides of the stream select and id inputs? Why not just group corresponding items (e.g: stream label+select, id label+input+play-button and latency label+input+set-button) in divs or fieldsets just using their contents sizes with some padding and margins and use a simple flow layout? E.g: Just showing them next to each other if it fits on a wide screen and wrapping those fieldsets into lines if necessary at smaller sizes. And meanwhile you can keep centering all of that horizontally as you are now if you like.
PS: I also don't understand why the latency is aligned to the right while the other inputs are aligned to the center. Is that so as to group the stream select with the id input? You could also use a fieldset (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) for that to label it accordingly.

Reply:
PS: Also please keep your white spacing clean. There's no space between margin-right: and auto. Think I've mentioned this before but I'd like to see prettier being used here to automatically keep formatting clean.